modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,54 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update Registry Certificate Authority with Private Registry
+
+. After you install the Multi Cluster Engine (MCE),  you must create a 'ConfigMap' in the MCE namespace with the private registry details. This ensures that you grant the hosted cluster workers the capability to retrieve images from the private registry.
```

Comment:
After the installation of MultiCluster Engine (MCE) operator, create 'ConfigMap' with the mirror registry details under the same namespace where MCE is installed .

Reply:
Fixed